pages/desktop/base.py

```diff
@@ -247,19 +259,29 @@ def more_menu(self, item=None):
 
     def click_developer_hub(self):
         self.find_element(*self._devhub_locator).click()
-        self.wait.until(EC.number_of_windows_to_be(2))
+        self.wait.until(
+            EC.number_of_windows_to_be(2),
+            message=f'Number of windows was {len(self.selenium.window_handles)}',
```

Comment:
could we add an "expected 2" at the end of this?
```suggestion
            message=f'Number of windows was {len(self.selenium.window_handles)}, expected 2',
```